pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -190,6 +208,38 @@ const (
 	KernelTunableModify KernelTunableBehavior = "modify"
 )
 
+// InitQOS creates the top level qos cgroup containers
+// We currently create containers for the Burstable and Best Effort containers
```

Comment:
We currently create containers for the Burstable and BestEffort classes or pods (not sure what you were intending)